pkgs/servers/amqp/rabbitmq-server/default.nix

```diff
@@ -7,11 +7,11 @@
 stdenv.mkDerivation rec {
   name = "rabbitmq-server-${version}";
 
-  version = "3.6.6";
+  version = "3.6.10";
 
   src = fetchurl {
-    url = "https://github.com/rabbitmq/rabbitmq-server/releases/download/rabbitmq_v3_6_6/rabbitmq-server-3.6.6.tar.xz";
```

Comment:
Why not stick with the GitHub url? 
`https://github.com/rabbitmq/rabbitmq-server/archive/rabbitmq_v3_6_10.tar.gz`

Reply:
mainly for convenience: the github url has 2 places for the version, one of which is not easy to interpolate